_toc.yml

```diff
@@ -0,0 +1,58 @@
+format: jb-book
+root: README
+
+parts:
+
+  - caption: About These Tutorials
+    chapters:
+    - file: doc/HowToUse                                                           # TODO
```

Comment:
Is this a doc that still needs to be developed?

Reply:
Yes... I've got a start there, but it does not cover all material in the repo, and makes no plans for placements of future documents.